quinn/src/runtime.rs

```diff
@@ -31,10 +31,15 @@ pub trait AsyncTimer: Send + Debug + 'static {
 
 /// Abstract implementation of a UDP socket for runtime independence
 pub trait AsyncUdpSocket: Send + Sync + Debug + 'static {
-    /// Send UDP datagrams from `transmits`, or register to be woken if sending may succeed in the
-    /// future
-    fn poll_send(&self, cx: &mut Context, transmits: &[Transmit])
-        -> Poll<Result<usize, io::Error>>;
+    /// Create a helper for awaiting I/O readiness
```

Comment:
Substantially expanded docs on all the new items. Is it clearer now?

Reply:
Yup, it's better. I still got a little confused because it's not that obvious that this increases waker state, but I'm not sure if there's a great fix for that.